File: Easy/palindromeNumber.py
class Solution(object):
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """
        isPal = False
        reversedString = ''
        
        for i in range(0, len(str(x))):
            reversedString = reversedString + str(x)[-i - 1]
        if reversedString == str(x):
            isPal = True
        return isPal

File: Easy/palindromeNumber.java
package Easy;
class Solution {
    public boolean isPalindrome(int x) {
        String str = String.valueOf(x);
        
        char[] strArr = str.toCharArray();
        
        String reversed = "";
        
        for (int i = str.length() - 1; i >= 0; i--)
        {
            reversed = reversed + strArr[i];
        }
        
        if (reversed.equals(str))
            return true;
        else
            return false;
    }
}